_content/booklets/buddhist-ceremonies-and-rituals-sri-lanka_kariyawasam-a-g-s.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Buddhist Ceremonies and Rituals of Sri Lanka"
+authors:
+  - "A.G.S. Kariyawasam"
+external_url: "https://www.buddhanet.net/pdf_file/ceremonies-srilanka6.pdf"
```

Comment:
This is fine for the external. But as this is a wheel, let's add the official BPS link as a source_url: https://www.bps.lk/library-search-select.php?id=wh402